lib/view/search_screen.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:acnhpedia/utility/constants.dart';
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class SearchScreen extends StatefulWidget {
+  const SearchScreen({Key? key}) : super(key: key);
+
+  @override
+  State<SearchScreen> createState() => _SearchScreenState();
+}
+
+class _SearchScreenState extends State<SearchScreen> {
+  final searchController = TextEditingController();
+  @override
+  void dispose() {
+    searchController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      debugShowCheckedModeBanner: false,
+      home: Scaffold(
+        backgroundColor: Colors.transparent,
+        body: Container(
+          constraints: const BoxConstraints.expand(),
+          decoration: const BoxDecoration(
+              image: DecorationImage(
+                  image: AssetImage("images/minimal_bg.png"),
+                  fit: BoxFit.cover)),
+          child: Center(
+            child: Padding(
+              padding: const EdgeInsets.only(top: 50.0),
+              child: Column(
+                children: [
+                  Text(
+                    'A C N H P E D I A',
+                    style: GoogleFonts.atma(
+                      color: Color(0xfffbfeb2),
+                      fontSize: 30,
+                    ),
+                  ),
+                  SizedBox(
+                    height: 200,
```

Comment:
why is height 200? doesn't make sense, it won't look the same across all devices